app/code/community/Bolt/Boltpay/Model/FeatureSwitch.php

```diff
@@ -35,7 +35,12 @@ class Bolt_Boltpay_Model_FeatureSwitch extends Bolt_Boltpay_Model_Abstract
     const ROLLOUT_KEY = 'rolloutPercentage';
 
     /**
-     * Determines if the Bolt server side has disabled the plugin. A false value will override local settings.
+     * @var string Feature switch that determines whether Git should be used as source for updates instead of Magento Connect
+     */
+    const BOLT_UPDATE_USE_GIT_SWITCH_NAME = 'M1_BOLT_UPDATE_USE_GIT';
```

Comment:
In which cases we need to check git instead of magento?

Reply:
@vitaliyreznikov 
This document covers the rationale behind the design:
https://docs.google.com/document/d/1Z67GkiGMfgtN_ejPMfWAzSSPKphIhUepPjOeWYVtLaI

![image](https://user-images.githubusercontent.com/973856/82032207-31556a00-969b-11ea-90b3-e6799a5efc13.png)

Reply:
I have a little concern here. Usually, we use Feature switches to roll out something. It's a way to test on few merchants or disable a new feature if we have any issue.
You use it in a slightly different way, but I think we can go ahead here.